src/main/kotlin/org/ionproject/integration/model/internal/DynamicObject.kt

```diff
@@ -0,0 +1,3 @@
+package org.ionproject.integration.model.internal
+
+data class DynamicObject(val jsonData: String, val textData: List<String>)
```

Comment:
Should we move this to ISELTimetable folder? At least for now it will only be used for that specific batch...